### PR TITLE
Construction Worker rebalance

### PR DESCRIPTION
**Pull request by TTerc**
[REWORK][BALANCE][INPUT WANTED]
## About the PR 

This PR is meant to be an alternative to #2523: Instead of removing construction worker completely, it tries to balance this role and makes it less overpowered. 

**List of changes:**
-Removes conworker's spacesuit, magboots, jetpack and breath mask
-Removes quite a few griefy floor textures and a LOT griefy wall textures from floor and wall planner (you can see the changes here: https://i.imgur.com/lVFpmZc.png) (and uh cave walls are back but I'm too lazy to screenshot them again)
-Removes conworker's material shaper (seriously, does anyone ever use that thing?)
-Lowers the RCDD capacity to 100
-Removes one of the matter cartridges from conworker's backpack: they start with one 100 capacity cartridge now, just so they can fill their RCDD
-Gives them mechlab access (lots of conworkers that actually like working on cool projects will appreciate this)
-Gives them EVA access as a reasonable alternative to the lack or roundstart spacesuit
-Gives them an engineering PDA

## Why's this needed? 
This PR will hopefully tackle the issues that people currently have with conworkers: It severly nerfs their RCDD and starting gear, hopefully discouraging them from breaking into the armory every single round and removes griefy textures from the floor planner (no more meaty floor spam, yay!) And mechlab access is a good enough incentive that should encourage conworkers to actually work on some constructions.

I wanted to make floor planner emaggable, but I don't think it is necessary, since conworkers cannot play as traitors anyway. If anyone wants to make it emaggable tho, old files that contain all the wall and floor textures are still there.

## Changelog

```changelog
(u)TTerc:
(*)Construction worker has been rebalanced! See PR for details.
```

### CODE REVIEW

**Comment by RSG250** [comment]
personally i would like if the cave textures were left in for the walls, but thats just me

**Comment by TTerc** [comment]
I think that's fair. They aren't as annoying and all over the place as the other walls

File: code/WorkInProgress/construction/tools.dm [hunk 1]
```diff
@@ -452,7 +452,7 @@
 
 	var/selecting = 0
 	var/mode = "floors"
-	var/icons = list("floors" = 'icons/turf/construction_floors.dmi', "walls" = 'icons/turf/construction_walls.dmi')
+	var/icons = list("floors" = 'icons/turf/conworker_floors.dmi', "walls" = 'icons/turf/conworker_walls.dmi')
```
Comment (ZeWaka) [inline code/WorkInProgress/construction/tools.dm:455]: don't like the naming of this, being tied to the job

Comment (TTerc) [reply code/WorkInProgress/construction/tools.dm:455]: hope the new ones are better

**Comment by Emcee-Gore** [comment]
I can see why one might wanna remove damaged plating, plating and catwalk floor textures.

But removing meat-station tiles is not something I am personally too happy about.
There are various use-cases for fun gimmicks that can be done with the meat-station floor. Spooky church. Lubed crusher hallway.

The EVA space-suit adjustment is fine, same with added Mechlab access. That is dope.

I still do not like that BOTH the starting volume of RCD matter-cartdriges AND the buff to removing walls and all that is gone.
How many walls can one now destroy with this to open up a shop-front where previously was a wall? 6 reinforced ones? The rest has to be done by hand or what?

That is my take on it at least.

**Comment by TTerc** [comment]
meat floor and walls are one of the most popular griefy textures that conworkers love spamming the station with so I'm still very much against it

but, I've thought a bit about it and I don't think bringing back the rcdd wall removal buffs is such a big deal so actually I'm willing to bring them back, if it's important for conworker players

**Pull request closed without merging**